pkg/storage/tsdb/bucketindex/markers.go

```diff
@@ -40,6 +42,10 @@ func NoCompactMarkFilenameMarkFilepath(blockID ulid.ULID) string {
 	return fmt.Sprintf("%s/%s-%s", MarkersPathname, blockID.String(), metadata.NoCompactMarkFilename)
 }
 
+func ConverterMarkFilePath(blockID ulid.ULID) string {
+	return fmt.Sprintf("%s/%s-%s", MarkersPathname, blockID.String(), parquet.ConverterMarkerFileName)
```

Comment:
Should we use a different marker path for parquet converter marker? Like `parquet-markers/<blockID>-<marker-name>`.

To avoid too many files inside the existing markers folder. It is also ok to use current path